fetch.bs

```diff
@@ -2049,6 +2052,34 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+
+<p>For <dfn export>determining the HTTP cache partition</dfn>, given <var>request</var>,
+run these steps:
+
+<ol>
+ <li>
+  <p>Let <var>environmentSettings</var> be <var>request</var>'s <a for=request>client</a>, if
+  present, otherwise return null.</p></li>
+
+ <li>
+  <p>Let <var>topLevelOrigin</var> be <var>environmentSettings</var>'s top-level origin and
+  <var>origin</var> be <a for="environment settings object">origin</a>.</p></li>
+
+ <li>
+  <p>Let <var>topLevelOriginAgentClusterKey</var> be the result of <span
+   data-x="obtain-agent-cluster-key">obtaining an agent cluster key</span>, given
```

Comment:
To cross link this, use `<a spec="HTML">obtaining an agent cluster key</a>`. Then we can make changes on the HTML side to allow you to remove the `spec="HTML"` (which is a way of overriding the fact that HTML currently doesn't export this term).

Reply:
Using <a spec> gave an error in make deploy, so I've changed it to <a href>